bundle.Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Use OpenShift golang builder image
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 AS builder
+
+WORKDIR /workspace
+
+COPY ./ ./
+RUN go mod download
+# needed for docker build but not for local builds
+RUN go mod vendor
+
+# Install operator-sdk
+RUN export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac) \
+OS=$(uname | awk '{print tolower($0)}') \
+OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.23.0; \
+curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}; \
+mv operator-sdk_${OS}_${ARCH} operator-sdk; \
+chmod +x operator-sdk
+
+# Set path to include local dir so standard make target can be used
+ENV PATH=$PATH:.
+
+# Setting VERSION here is workaround because he builder image sets VERSION to the golang version
+RUN VERSION=1.3.1 make bundle
+
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=sandboxed-containers-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.19.0+git
```

Comment:
we're using operator-sdk 1.23 above to build. I don't think it has any effect but we could update it here to 1.23 as well.

Reply:
Depending on the outcome of [my earlier comment](https://github.com/openshift/sandboxed-containers-operator/pull/219/files/3821f2a206321b806f4179cf6ae323cbb13daf7b#r989812670) this change might be made on `bundle-custom.Dockerfile` instead.